### PR TITLE
Fix/event list

### PR DESCRIPTION
**Pull request by DiD3n**
This PR is collection of improvments more or less related to EventList component

### CODE REVIEW

File: src/components/ui/EventList/EventList.tsx [hunk 1]
```diff
@@ -13,14 +14,21 @@ import { EventListSkeleton } from "../../loaders/Skeletons/EventListSkeleton/Eve
 import { PlaceholderSwitcher } from "../../utils/animations/PlaceholderSwitcher";
 import { AnimatedList } from "../../utils/animations/AnimatedList";
 
+export interface EventListProps {
+    page?: number
+    size?: number
```
Comment (JUtupe) [inline src/components/ui/EventList/EventList.tsx]: Maybe pagination should be handled inside EventList component? Is there need to handle pagination in every use of EventList?

File: src/views/studio/StudioEventList/StudioEventList.tsx [hunk 2]
```diff
@@ -39,23 +24,7 @@ export const StudioEventList: React.FC = () => {
                 </Link>
             </div>
 
-            {data && data.events
-                .map(Event.fromData)
-                .map(event =>
-                    <Link
-                        key={event.id}
-                        style={{textDecoration: 'none'}}
-                        to={`/studio/events/${event.vanityUrl}`}>
```
Comment (JUtupe) [inline src/views/studio/StudioEventList/StudioEventList.tsx:48]: This link differs from EventList link, move it to props so it can be reusable.

**Review by JUtupe — CHANGES_REQUESTED** [review]

File: src/views/studio/StudioEventList/StudioEventList.tsx (hunk 2 above)
Comment (DiD3n) [reply src/views/studio/StudioEventList/StudioEventList.tsx:48]: 👍 
no :)

Comment (DiD3n) [reply src/views/studio/StudioEventList/StudioEventList.tsx:48]: jk, it's done mate

**Review by JUtupe — APPROVED** [review]

**Pull request merged**